src/components/Dashboard/Pay/index.tsx

```diff
@@ -90,6 +88,7 @@ export default function Pay() {
         </Tooltip>
       )
     } else {
+      console.log('currentFC.configured.eq(0): ', currentFC.configured.eq(0))
```

Comment:
help i am a log and i am trapped in this diff

Reply:
I got you dawg